test/accessibility/apps/dashboard.ts

```diff
@@ -49,9 +58,115 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
       await a11y.testAppSnapshot();
     });
 
+    it('Open Edit mode', async () => {
+      await PageObjects.dashboard.switchToEditMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Open add panel', async () => {
+      await dashboardAddPanel.clickOpenAddPanel();
+      await a11y.testAppSnapshot();
+    });
+
+    it('add a visualization', async () => {
+      await testSubjects.click('savedObjectTitle[Flights]-Delay-Buckets');
+      await a11y.testAppSnapshot();
+    });
+
+    it('add a saved search', async () => {
+      await dashboardAddPanel.addSavedSearch('[Flights] Flight Log');
+      await a11y.testAppSnapshot();
+    });
+
+    it('save the dashboard', async () => {
+      await PageObjects.dashboard.saveDashboard(dashboardName);
+      await a11y.testAppSnapshot();
+    });
+
+    it('Open Edit mode', async () => {
+      await PageObjects.dashboard.switchToEditMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('open options menu', async () => {
+      await PageObjects.dashboard.openOptions();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Should be able to hide panel titles', async () => {
+      await testSubjects.click('dashboardPanelTitlesCheckbox');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Should be able display panels without margins', async () => {
+      await testSubjects.click('dashboardMarginsCheckbox');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Open add panel', async () => {
+      await dashboardAddPanel.clickOpenAddPanel();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Add one more saved object to cancel it', async () => {
+      await testSubjects.click('savedObjectTitle[Flights]-Average-Ticket-Price');
+      await a11y.testAppSnapshot();
+    });
+
+    it('Close add panel', async () => {
+      await dashboardAddPanel.closeAddPanel();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Exit out of edit mode', async () => {
+      await PageObjects.dashboard.clickCancelOutOfEditMode(dashboardName);
+      await a11y.testAppSnapshot();
+    });
+
+    it('Discard changes', async () => {
+      await PageObjects.common.clickConfirmOnModal();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Test full screen', async () => {
+      await PageObjects.dashboard.clickFullScreenMode();
+      await a11y.testAppSnapshot();
+    });
+
+    it('Exit out of full screen mode', async () => {
+      const logoButton = await PageObjects.dashboard.getExitFullScreenLogoButton();
+      await logoButton.moveMouseTo();
+      await PageObjects.dashboard.clickExitFullScreenTextButton();
```

Comment:
I have seen the same 3 lines of code in one of the `full_screen_mode.js` tests.
How about we wrap it up in a function placing in `dashboard_page.ts`:
```
async function exitFullScreenMode() {
  const logoButton = await PageObjects. dashboard.getExitFullScreenLogoButton();
  await logoButton.moveMouseTo();
  await PageObjects.dashboard.clickExitFullScreenTextButton();
}
```

If you agree, please update the other test as well